ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/ledger/api/tls/TlsConfigurationTest.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.tls
+
+import org.scalatest.WordSpec
+import java.security.Security
+
+import org.scalatest.Matchers
+
+class TlsConfigurationTest extends WordSpec with Matchers {
+  "TlsConfiguration" should {
+    "set OCSP JVM properties" in {
+      disableChecks()
+      verifyOCSP(DISABLED)
+
+      TlsConfiguration.Empty
+        .copy(
+          enabled = true,
+          revocationChecks = true
+        )
+        .setJvmTlsProperties()
+
+      verifyOCSP(ENABLED)
+    }
+  }
+
+  private val sslCheckRevocationProperty: String = "com.sun.net.ssl.checkRevocation"
+  private val ocspProperty: String = "ocsp.enable"
+  private val ENABLED = "true"
+  private val DISABLED = "false"
+
+  private def disableChecks(): Unit = {
```

Comment:
Please move these into a separate (non-test) object, i.e., the logic to enable vs. disable JVM specific properties should be in one place (see comment on `TlsConfiguration`).